### PR TITLE
Add:updated documentation for Context.

### PR DESCRIPTION
**Pull request by xonx4l**
issue-:#88

/claim #88

updated documentation for Context.

Added information about using-:

Environment variables
Variables
mustache templates

### CODE REVIEW

**Review by tusharmath — APPROVED** [review]

File: docs/guides/context.md [hunk 1]
```diff
@@ -67,17 +67,58 @@ type Post {
...
-  users: [User]! @http(baseUrl: "{{env.API_ENDPOINT}}", path: "/users")
+Setting the base URL for API requests:
+```type Query {
+  users: [User]! @http(baseUrl: "{{env.API_ENDPOINT}}", path: "/users") 
 }
 ```
 
-In the above example, `env.API_ENDPOINT` refers to an environment variable called API_ENDPOINT, which should be defined in your server settings.
+Toggling features on/off:
+```type Query {
```
Comment (tusharmath) [inline docs/guides/context.md:81]: Use the following syntax.

```
```graphql
type Query
```

```

Currently it's 

```
```type Query
```

```

File: docs/guides/context.md [hunk 2]
```diff
@@ -67,17 +67,58 @@ type Post {
...
+}
+```
+Variables have the following properties:
+
+->Declared with $name syntax
+->Type needs to be provided
+->Passed as separate parameter to query
+->Can be used in place of literal values
+->Help avoid duplication in queries
+->Enable dynamic query construction.
```
Comment (tusharmath) [inline docs/guides/context.md:121]: This looks incomplete.

Comment (xonx4l) [reply docs/guides/context.md:125]: @tusharmath added some more features info. You can review . Thanks

**Comment by tusharmath** [comment]
Closing this PR because it talks about features that don't exist.

**Pull request closed without merging**